tests/test-history.sh

```diff
@@ -16,6 +16,18 @@ echo "1..1"
 HISTORY_START_TIME=$(date +"%Y-%m-%d %H:%M:%S")
 sleep 1
 
+if ! logger "Checking whether Flatpak can use the journal..."; then
+    echo "ok 1 # SKIP Cannot write to Journal with logger"
```

Comment:
maybe slightly better to use the `skip` function from `libtest.sh`

Reply:
Sure, done.

Reply:
Ah, I remember now why I didn't do this. `skip` can only be used before the test plan `1..1` is written.

I've reordered it to use `skip`.